### ISSUE TITLE
Errors that I don't understand...

### ISSUE THREAD

**Issue by Undefined-Variables** [pH7Software/Social-Dating-Android-App#3]
I am trying to build an android app using AndroidStudio 3.6 by following the basic instructions in the README but I am hitting the same error no matter what I do. 

I have zero app building experience but have a nice website running on pH7cms so would like to create an android and ios app to offer more choice in the way the website is accessed. Can you explain how to solve this please?
![Capture](https://user-images.githubusercontent.com/52648055/78462025-2d125600-76c6-11ea-89b2-3e16bcf8a2cb.PNG)
  
Is this just the machine im trying to build with having AMD chipset?

**Comment by ultra-pool** [comment]
Doesn't seem like the main dev supports this any more. It been a very time since any updates have been done. This app was made using android sdk 23 and hasn't been updated for the newer sdk's.

**Comment by Undefined-Variables** [comment]
Oh that is a shame, it sounded like a dream solution to getting your website in to an app. Does make me wonder if there are any other projects trying to do the same. Going to have to get myself some reading material and learn some java... Android application style.

Thanks for your reply :)

**Comment by ultra-pool** [comment]
I was just trying to update it myself. Got alot of the errors fixed. But its down to 2 error's with the newer sdk update.

**Comment by Undefined-Variables** [comment]
Have you got this in a repository? I know pH7cms pretty well and would like to have a look at your version if you don't mind?

**Comment by ultra-pool** [comment]
Yeah finding nothing that helps fixes all the problems on this code has. Just tons of trash. And same for the main site code. Ill write this off as time wasted and trying to something that works atleast halfa$$

**Comment by Undefined-Variables** [comment]
I have been working on this software for a long time and it is not crap you just have to understand how it works to be able to make the most out of it and I don't mean somebody who is able to install php scripts that have had blockbuster money spent on development. Anybody can throw a word press site that is their strategy to make things as simple as possible so the masses can use it. 

I wouldn't get in an AirBus380 and say that is a pile of shit because my opinion would be based on my lack of ability to fly and operate it. This software is an honest attempt to provide a building block to get you started but you have to know how to build it in to what you require. 

Going back to the Android App. There is nothing special about it, there are many up to date view point applications that do the same thing. Its like taking a short cut because its not a real app in the traditional sense. Its a portal that uses your same website. IMO an app should run off its own resources and be able to be used without internet access. People who take shortcuts get no where in the long run. 

My test site https://undefined-variables.com/ph7cms was installed yesterday using the same software available to you and the only changes have been in the messaging system everything else is standard and it has no issues at all. It begins a long time before software. The server needs to be perfectly tuned for the job you are expecting of it. Slating peoples hard work because you don't have the mental ability to use it is not a good look!

**Comment by dougwilson0000** [comment]
Have you tried to turn on passive HTTPS in the extensions? 

This gives the impression of a web app and it even gives its own icon on mobile!
